### ISSUE TITLE
new rule - unused `rumdl-disable-line` comment

### ISSUE THREAD

**Issue by DetachHead** [rvben/rumdl#733]
### Problem or motivation

thanks for the quick fixes for the issues i opened yesterday! it would be nice if when i updated rumdl, it would warn me about the `rumdl-disable-line` comments in my project that are no longer necessary now that those issues have been resolved

### Proposed solution

a new rule called `unused-disable-comment` or something

### Feature category

New lint rule

### Example markdown

```markdown
1. no error here

!!! example

    1. no error here.
    2. false positive which has been fixed in 0.2.32 <!-- rumdl-disable-line blanks-around-lists -->
    3. no error here
```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
Shipped in v0.2.48 as MD087, `unused-disable-comment`. It is opt-in, since a comment that is dead in one checkout can be load-bearing under a different rule set:

```toml
[global]
extend-enable = ["MD087"]
```

or `rumdl check --extend-enable MD087` for a one-off sweep. It reports the comment that suppresses nothing (`Unused disable-line comment: MD013`) and stays silent on one still doing work.

One gap in 0.2.48 worth flagging: a disable comment inside an indented container body (a MkDocs admonition or content tab) is honoured but not judged, so a dead one there goes unreported. Fixed on main, ships in the next patch.

Thanks @DetachHead. The comments left behind after a fix are exactly the ones nobody goes looking for.
